front/app/containers/IdeasIndexPage/index.tsx

```diff
@@ -77,8 +77,8 @@ const PageTitle = styled.h1`
 export interface QueryParameters {
```

Comment:
Putting these into the api folder would be clean-up too. I wouldn't do it in this PR. Maybe you can just create a ticket clean-up for TAN-3025 (in which we can create child tickets), so we have a PR that lets us easily track the work we need to do.